pages/docs/couple-your-code/couple-your-code-moving-meshes.md

```diff
@@ -2,7 +2,79 @@
 title: Dealing with moving meshes
 permalink: couple-your-code-moving-meshes.html
 keywords: api, adapter, ALE, geometries, CFD
-summary: "TODO"
+summary: "preCICE supports ALE-methods, direct-mesh-access, and pseudo-meshes to handle most scenarios of moving meshes. A remeshing API is on the roadmap and being actively developped."
 ---
 
-TODO
+## Static reference domain
+
+preCICE supports static meshes, meaning dynamic meshes can still be emulated using node displacements as described by the Arbitrary-Lagrangian-Eulerian or ALE-methods.
+
+Biggest downside of this method is that spatially aware mapping-schemes in preCICE work in the reference domain, not the physical domain.
```

Comment:
I would not necessarily call this a downside.

Reply:
```suggestion
Note that spatially aware mapping-schemes in preCICE work in the reference domain, not the physical domain.
```